### PR TITLE
UCS/VFS: Fixed type of UCP worker features in VFS.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Fixed crash while reading num_all_eps and keepalive/ep_count.
Changed primitive type to `UCS_VFS_TYPE_U32` (which is equal to `UCS_VFS_TYPE_UNSIGNED | UCS_VFS_TYPE_INT`).
`UCS_VFS_TYPE_UNSIGNED` should not be used as a parameter for `ucs_vfs_obj_add_ro_file`, because this is a type modifier.

Added assert to default section in `ucs_vfs_show_primitive` and dumped error to string buffer to avoid crash in such cases in future.

### CODE REVIEW

File: src/ucs/vfs/base/vfs_obj.c [hunk 1]
```diff
@@ -121,6 +121,8 @@ void ucs_vfs_show_primitive(void *obj, ucs_string_buffer_t *strb, void *arg_ptr,
             ulvalue = *(unsigned long*)arg_ptr;
             break;
         default:
+            ucs_assertv(0, "incorrect primitive type");
```
Comment (yosefe) [inline src/ucs/vfs/base/vfs_obj.c:124]: maybe ucs_warn() with more details (name, ptr, type, etc)

**Comment by brminich** [comment]
maybe rename `UCS_VFS_TYPE_UNSIGNED` and `UCS_VFS_TYPE_HEX` in a way that they will not be confused with types?
like `UCS_VFS_TYPE_MODIFIER_UNSIGNED`?

**Comment by yosefe** [comment]
> UCS_VFS_TYPE_MODIFIER_UNSIGNED

`UCS_VFS_TYPE_FLAG_UNSIGNED` ?

**Review by brminich — APPROVED** [review]

File: src/ucs/vfs/base/vfs_obj.c [hunk 2]
```diff
@@ -121,12 +121,14 @@ void ucs_vfs_show_primitive(void *obj, ucs_string_buffer_t *strb, void *arg_ptr,
             ulvalue = *(unsigned long*)arg_ptr;
             break;
         default:
+            ucs_warn("Incorrect type %lu of arg_ptr %p", arg_u64, arg_ptr);
```
Comment (yosefe) [inline src/ucs/vfs/base/vfs_obj.c:124]: 1. smallcase
2. can we print the vfs path?
`ucs_warn("vfs object %p (%s): incorrect type 0x%lx of arg_ptr %p", obj, path, arg_u64, arg_ptr)`

Comment (rakhmets) [reply src/ucs/vfs/base/vfs_obj.c:124]: only if we pass additional argument to text callback.
currently it is `(*ucs_vfs_file_show_cb_t)(void *obj, ucs_string_buffer_t *strb, void *arg_ptr, uint64_t arg_u64)`

Comment (yosefe) [reply src/ucs/vfs/base/vfs_obj.c]: > only if we pass additional argument to text callback.
> currently it is `(*ucs_vfs_file_show_cb_t)(void *obj, ucs_string_buffer_t *strb, void *arg_ptr, uint64_t arg_u64)`

for the path? i guess we could find path by obj ptr, or you wanted to avoid it?

Comment (rakhmets) [reply src/ucs/vfs/base/vfs_obj.c]: We will get path to directory representing the object, not path to file where we get the issue.
Maybe it is better than nothing :)

Comment (yosefe) [reply src/ucs/vfs/base/vfs_obj.c]: right, maybe reword:
ucs_warn("vfs object %p (%s) attribute %p: incorrect type 0x%lx of arg_ptr %p", obj, path, arg_ptr, arg_u64)

Comment (yosefe) [reply src/ucs/vfs/base/vfs_obj.c]: ok, so let's print ptr only, no path

**Review by yosefe — APPROVED** [review]
